docs/howtos/DataPlane Generation - DPG/08methodInputs.mdx

```diff
@@ -773,3 +775,171 @@ public void register(Schema schema);
 
 </TabItem>
 </Tabs>
+
+### Using Azure.Core.ResourceOperations template
+
+Resource create and update operations are not impacted by spread since they all have explicit defined body parameter.
+Only resource action operations are impacted by spread.
+
+If the action parameter is a model, then the model will be spread.
+
+<Tabs>
+<TabItem value="typespec" label="TypeSpec" default>
+
+```typespec
+@resource("widgets")
+model Widget {
+  @key("widgetName")
+  name: string;
+}
+
+model RepairInfo {
+  problem: string;
+  contact: string;
+}
+
+model RepairResult {
+  reason: string;
+  info: string;
+}
+
+alias Operations = Azure.Core.ResourceOperations<{}>;
+
+op scheduleRepairs is Operations.ResourceAction<Widget, RepairInfo, RepairResult>;
+```
+
+</TabItem>
+
+<TabItem value="python" label="Python">
+For Python, we will also generate the overloads described in the Http Post section, but omitting for brevity
+
+```python
+class RepairInfo:
+  problem: str
+  contact: str
+
+class RepairResult:
+  reason: str
+  info: str
+
+def scheduleRepairs(self, problem: str, contact: str, *, content_type: str = "application/json") -> RepairResult:
```

Comment:
You seems missed the `widgetName`

https://cadlplayground.z22.web.core.windows.net/cadl-azure/?c=aW1wb3J0ICJAdHlwZXNwZWMvcmVzdCI7CskZYXp1cmUtdG9vbHMvyCUtxhVjb3JlIjsKCnVzaW5nIFR5cGVTcGVjLkh0dHA70BVSZXN00RVWZXJzaW9uaW5nyBtBxFkuQ29yZTsKCkBzZXJ2aWNlKHsKICB0aXRsZTogIkFwcCIsCn0pCkB2xj5lZChBcGnHS3MpCm5hbWVzcGFjZSBDYWRsLkFwcDsKCmVudW0gyycgxFNAdXNlRGVwZW5kZW5jeShnbG9iYWwu6gCD6ACgcy52MV8wX1ByZXZpZXdfMikKICB2MjAyMl8wNl8wMV9wxhk6ICLEFS0wNi0wMS3HFeQArAoKQHJlc291cmNlKCJ3aWRnZXRzIikKbW9kZWwgV8UQ5gCQa2V5yCFOYW1lIsRp5ADPOiBzdHLlASZ9xzZSZXBhaXJJbmZvxTpwcm9ibGVtyiggIGNvbnRhY3TYO1Jlc3VsxndyZWFzb27MPGluZm%2FMOWFsaWFzIE9wZXJhdOUBPj3rAankAdvlANzKIDx7fT47Cm9wIHNjaGVkdWxlxnRzIGnMRsk5QWPEEDzmAQos6wDcyAzmAK0%2BOwo%3D&e=%40azure-tools%2Ftypespec-autorest&options=%7B%7D